src/main.rs

```diff
@@ -101,9 +101,15 @@ fn main() {
         .add_sensor(TemperatureSensorTemplate {
             unit: "°C".into(),
             location: "Hackerspace".into(),
-            name: Some("Room Temperature".into()),
+            name: Some("Room Temperature (Entrance)".into()),
             description: None,
-        }, "temperature_room".into())
+        }, "temperature_entrance".into())
+        .add_sensor(TemperatureSensorTemplate {
+            unit: "°C".into(),
+            location: "Hackerspace".into(),
+            name: Some("Room Temperature (Tables)".into()),
+            description: None,
+        }, "temperature_tables".into())
```

Comment:
I don't like "tables" because of the ambiguity, but I don't have an idea for a better name. If not "chairs" ;)

Reply:
"backoffice" (because it's in the back of the room (as seen from the entrance) and it has tables with laptops or other "work" stuff) :wink: